go.mod

```diff
@@ -5,9 +5,9 @@ go 1.21
 toolchain go1.21.0
 
 require (
-	chainguard.dev/apko v0.10.1-0.20230906171757-7c910a4b75ca
+	chainguard.dev/apko v0.10.1-0.20231017165019-27c6d86b09e6
```

Comment:
Can we bump to 0.11 and cut 0.11 of tf-apko?

Reply:
I'm lazily avoiding the digest change but we can do that.

Reply:
I don't think I can without the digest change for some reason.